src/gui/sdl_gui.cpp

```diff
@@ -1770,6 +1770,7 @@ void GFX_Init()
 	sdl.draw.render_height_px = FallbackWindowSize.y;
 
 	create_window_and_renderer();
+	SDL_RaiseWindow(sdl.window);
```

Comment:
Suggest raising the window inside the `create_window_and_renderer()` function and putting a comment that's it a Mac problem. I've not seen this be an issue on Windows or Linux in the past.

Tested on Linux just now with this branch and there's no behavior change. Seems fine to explicitly raise the window on all platforms anyway but a comment would be good so a non-Mac developer doesn't remove this in the future.